app/routes/order/international_order.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+import AuthorizeRoute from "./../authorize";
+
+export default AuthorizeRoute.extend({
+  orderService: Ember.inject.service(),
+
+  async model() {
+    return {
+      shipmentSubsequentCode: await this.get(
+        "orderService"
+      ).fetchShipmentorCarryoutCode("Shipment")
```

Comment:
"Shipment" and "Carrout" are also better placed in a constant, since the string is referred multiple places